plugins/mcreator-localization/help/default/entity/visual_scale.md

```diff
@@ -0,0 +1,3 @@
+This feature allows you to control the size of the entity in-game.

```

Comment:
I'd say **visible size** and omit _in-game_ as it may mean BB scale too (in the other tip the opposite is implicitly prooved).